utils/types.go

```diff
@@ -0,0 +1,7 @@
+package utils
+
+type BatchKey struct {
```

Comment:
Why do we need a separate struct for this? Why can't `CreateBatchKeys` just return a list of normal `Keys` (or whatever the other struct is called)?

Reply:
What other struct. we are returning array of `BatchKey` so that we can read each keys password and hex and encrypted file.